packages/theme/src/Theme/component-changes.tsx

```diff
@@ -982,12 +995,17 @@ export const Tooltip = (mode: PaletteMode, colors: MaskColor) => ({
             },
             styleOverrides: {
                 tooltip: {
+                    maxWidth: 300,
                     padding: 10,
                     fontSize: 14,
                     borderRadius: 4,
                     lineHeight: '18px',
                     backgroundColor: colors.maskColor.tips,
                     color: colors.maskColor.bottom,
+                    '& > *': {
```

Comment:
Avoid using `*`.

Reply:
But the `*` is needed here, because there is no way to determine what components will be included in the tooltip.